.ebextensions/change_timezone.config

```diff
@@ -0,0 +1,3 @@
+commands:
+  00_change_timezone:
+    command: ln -sf /usr/share/zoneinfo/America/Los_Angeles /etc/localtime
```

Comment:
I don't know the implication and if this will fix the OAI problem. Should we get a repro of the problem in stage or another EB instance to find out what happens with the time change.

Reply:
EB didn't like this exact config file.  It was saying that there was an "invalid key". I removed all the whitespace and re-added it and I think that solved that problem.  Also, this command wasn't actually setting the timezone so I found another command that seems to work.  Now I got a successful harvest in my test environment.